detections/endpoint/windows_credentials_access_via_vaultcli_module.yml

```
@@ -0,0 +1,69 @@
+name: Windows Credentials Access via VaultCli Module
+id: c0d89118-3f89-4cd7-8140-1f39e7210681
+version: 1
+date: '2024-11-29'
+author: Teoderick Contreras, Splunk
+data_sources:
+- Sysmon Event ID 7
+type: Anomaly
+status: production
+description: The following analytic detects the presence of information-stealing malware that exploits the VaultCLI.dll module to extract credentials from the Windows Credential Vault. It identifies abnormal interactions with VaultCLI.dll, particularly those initiated by processes located in publicly writable Windows folder paths. The analytic monitors suspicious API calls, unauthorized credential access patterns, and anomalous process behaviors indicative of malicious activity. By leveraging a combination of signature-based detection and behavioral analysis, it effectively flags attempts to misuse the vault for credential theft, enabling swift response to protect sensitive user data and ensure system security.
```

Made small changes to the description in order to make it describe the detection logic more accurately 

```suggestion
description: The following analytic detects potentially abnormal interactions with VaultCLI.dll, particularly those initiated by processes located in publicly writable Windows folder paths. The VaultCLI.dll module allows processes to extract credentials from the Windows Credential Vault. It was seen being abused by information stealers such as Meduza. The analytic monitors suspicious API calls, unauthorized credential access patterns, and anomalous process behaviors indicative of malicious activity. By leveraging a combination of signature-based detection and behavioral analysis, it effectively flags attempts to misuse the vault for credential theft, enabling swift response to protect sensitive user data and ensure system security.
```